controllers/main.go

```diff
@@ -2119,6 +2160,220 @@ func (controller *MainController) IsValidVoteBits(userVoteBits uint16) bool {
 	return userVoteBits&^usedBits == 0
 }
 
+func (controller *MainController) APIv3GetPubKey(c web.C, r *http.Request) *poolapi.GetPubKeyResponse {
+	return &poolapi.GetPubKeyResponse{
+		Timestamp: time.Now().Unix(),
+		PubKey:    controller.Cfg.PubKey,
+	}
+}
+
+func (controller *MainController) APIv3GetFee(c web.C, r *http.Request) *poolapi.GetFeeResponse {
+	return &poolapi.GetFeeResponse{
+		Timestamp: time.Now().Unix(),
+		Fee:       controller.Cfg.PoolFees,
```

Comment:
I believe this should be returning an absolute value, the exact fee for the ticket specified in the request.

Reply:
should **not** be returning an absolute value, you mean?

Reply:
Yep, you're right